doc/mediasdk-man.md

```diff
@@ -7872,6 +7880,37 @@ This structure is used to pass decryption status report index for Common Encrypt
 
 This structure is available since SDK API 1.30.
 
+## <a id='mfxExtPartialBitstreamParam'>mfxExtPartialBitstreamParam</a>
+
+**Definition**
+
+```C
+typedef struct {
+    mfxExtBuffer    Header;
+    mfxU32          BlockSize;        /* output block granulatiry for Granularity = MFX_PARTIAL_BITSTREAM_BLOCK */
+    mfxU16          Granularity;      /* granulatiry of the partial bitstream: slice/block/any */
+    mfxU16          reserved[8];
+} mfxExtPartialBitstreamParam;
+```
+
+**Description**
+
+This structure is used by AVC HW encoder to output parts of bitstream as soon as they ready. The application can attach this extended buffer to the [mfxVideoParam](#mfxVideoParam) structure at init time.
```

Comment:
> by AVC HW encoder

Actually structure might apply to any codec. Moreover, below you suggest to Query and check whether codec supports the feature or not. I think it is better to remove any mentioning of the specific codec (AVC) from this text.

Reply:
fixed in new update of pull request